stable/anchore-engine/templates/enterprise_ui_secret.yaml

```diff
@@ -15,9 +15,9 @@ metadata:
 type: Opaque
 stringData:
   {{- if .Values.anchoreGlobal.dbConfig.ssl }}
-  ANCHORE_APPDB_URI: 'postgresql://{{ index .Values "postgresql" "postgresUser" }}:{{ index .Values "postgresql" "postgresPassword" }}@{{ template "db-hostname" . }}/{{ index .Values "postgresql" "postgresDatabase" }}?ssl=verify-full'
+  ANCHORE_APPDB_URI: 'postgresql://{{ ternary (index .Values "anchoreEnterpriseUi" "dbUser") (index .Values "postgresql" "postgresUser") (hasKey .Values.anchoreEnterpriseUi "dbUser" ) }}:{{ index .Values "postgresql" "postgresPassword" }}@{{ template "db-hostname" . }}/{{ index .Values "postgresql" "postgresDatabase" }}?ssl=verify-full'
```

Comment:
@brandtkeller if `.Values.anchoreEnterpriseUi.dbUser` is set, should we be handling the password separately as well? Using `.Values.postgresql.postgresPassword`, which is shared with the engine db user, doesn't seem ideal.

Reply:
I believe that is a fair assessment. We're trying to establish some ability to separate the users and their credentials - it would make sense to allow a non-shared password.

Reply:
Added support for both user/password to be unique. It's a lengthy template with multiple ternaries - so i'm open to input or anyone doing direct modification.